src/ui/Editor.ts

```diff
@@ -242,7 +241,7 @@ export function Editor() {
       {
         reset: {
           value: false,
-          onChange: () => setVehicleEditor({ debug: false, reset: false, ...vehicleConfig, ...filteredWheelInfo }),
```

Comment:
This reset is for the state of the editor, not for the game, it shouldn't be changed.